spec/requests/create_payment_on_successful_charge_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.